modules/deploy/pages/deployment-option/cloud/api/cloud-api-quickstart.adoc

```diff
@@ -35,8 +35,33 @@ rpk cloud byoc aws apply --redpanda-id=<metadata.cluster_id>
 Serverless::
 +
 --
-. xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[Authenticate to the API].
+. In the Redpanda Cloud UI, create a https://cloud.redpanda.com/clients[service account (client)] and use the client ID and secret to request an API token. Use the token to xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[authenticate to the API].
+. Make a `GET /v1beta2/resource-groups` request to retrieve the default resource group ID.
++
+[,bash]
+----
+curl -H 'Authorization: Bearer <token>' https://api.redpanda.com/v1beta2/resource-groups
+----
+. Make a `GET /v1beta2/serverless/regions` request to see available regions.
++
+[,bash]
+----
+curl -H 'Authorization: Bearer <token>' https://api.redpanda.com/v1beta2/serverless/regions?cloud_provider=CLOUD_PROVIDER_AWS
```

Comment:
Single quotes are needed around the last argument (the URL) because otherwise the question mark will be interpreted as a shell character instead of as part of the URL.

One other thing, we may want to note somewhere that if the bearer token is filled in with a shell substitution variable (e.g. curl -H "Authorization: Bearer **$TOKEN**" https://api.redpanda.com/v1beta2/resource-groups), then that argument to the command needs to be surrounded by double quotes rather than single quotes. This is a well known thing but people may forget when copy/pasting commands. This is less important but just a thought.

Reply:
Thanks @ajyoung , I have exactly the same suggestion